subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftDepsHandler.java

```diff
@@ -83,8 +84,14 @@ private void write(File moduleSwiftDeps, final SwiftDeps swiftDeps) {
         IoActions.writeTextFile(moduleSwiftDeps, new Action<BufferedWriter>() {
             @Override
             public void execute(BufferedWriter bufferedWriter) {
-                Yaml yaml = new Yaml();
-                yaml.dump(swiftDeps, bufferedWriter);
+                PrintWriter pw = new PrintWriter(bufferedWriter);
+                pw.println("version: \"" + swiftDeps.version + "\"");
+                pw.println("options: \"" + swiftDeps.options + "\"");
+                pw.println("build_time: [" + swiftDeps.build_time.get(0) + ", " + swiftDeps.build_time.get(1) + "]");
+                pw.println("inputs:");
+                swiftDeps.inputs.forEach((file, timestamp) -> {
+                    pw.println("  \"" + file + "\": [" + timestamp.get(0) + ", " + timestamp.get(1) + "]");
+                });
```

Comment:
Looks `yaml` to me. Can we get a comment here on why we do raw `println`s here?

Reply:
@big-guy and I couldn't get the YAML generator to spit out the right file output